cirrus/server/cirrus/experiment_recipes.py

```diff
@@ -18,38 +18,62 @@ class RecipeType(Enum):
+        self.sdk_preview = sdk_preview
 
     def get_recipes(self) -> dict[str, list[Any]]:
         return self.recipes
 
+    def get_preview_recipes(self) -> dict[str, list[Any]]:
+        return self.preview_recipes
+
     def get_recipe_type(self, experiment_slug: str) -> str:
-        for experiment in self.get_recipes()["data"]:
-            if experiment["slug"] == experiment_slug:
-                if experiment.get("isRollout", False):
-                    return RecipeType.ROLLOUT.value
-                else:
-                    return RecipeType.EXPERIMENT.value
+        return self._get_recipe_type(self.get_recipes(), experiment_slug)
+
+    def get_preview_recipe_type(self, experiment_slug: str) -> str:
+        return self._get_recipe_type(self.get_preview_recipes(), experiment_slug)
 
+    def _get_recipe_type(
+        self, recipes: dict[str, list[Any]], experiment_slug: str
+    ) -> str:
+        for experiment in recipes["data"]:
+            if experiment["slug"] == experiment_slug:
+                return (
+                    RecipeType.ROLLOUT.value
+                    if experiment.get("isRollout", False)
+                    else RecipeType.EXPERIMENT.value
+                )
         return RecipeType.EMPTY.value
 
-    def update_recipes(self, new_recipes: dict[str, list[Any]]) -> None:
-        self.recipes = new_recipes
-        self.sdk.set_experiments(json.dumps(self.recipes))
+    def fetch_recipes(self) -> None:
+        self._fetch_and_update(self.url, self.update_recipes)
+        self._fetch_and_update(self.preview_url, self.update_preview_recipes)
 
-    def fetch_recipes(self):
+    def _fetch_and_update(
+        self, url: str, update_method: Callable[[dict[str, Any]], None]
+    ) -> None:
         try:
-            response = requests.get(self.url)
-            response.raise_for_status()  # raises an HTTPError for bad status codes
+            response = requests.get(url)
+            response.raise_for_status()
+            data = response.json().get("data")
+            if data is not None:
+                update_method({"data": data})
+                logger.info(f"Fetched resources: {data}")
+            else:
+                logger.warning("No recipes found in the response")
         except requests.exceptions.RequestException as e:
             logger.error(f"Failed to fetch recipes: {e}")
             raise e
-        data = response.json().get("data")
-        if data is not None:
-            self.update_recipes({"data": data})
-            logger.info(f"Fetched resources: {data}")
-        else:
-            logger.warning("No recipes found in the response")
+
+    def update_recipes(self, new_recipes: dict[str, list[Any]]) -> None:
+        self.recipes = new_recipes
+        self.sdk_live.set_experiments(json.dumps(self.recipes))
+
+    def update_preview_recipes(self, new_recipes: dict[str, list[Any]]) -> None:
+        self.preview_recipes = new_recipes
```

Comment:
Oh same thing here, if we're going to store the recipes right on the object for both the live and preview collections we should probably follow teh same naming convention we do for the sdks.